great_expectations/experimental/rule_based_profiler/rule/rule.py

```diff
@@ -141,7 +140,7 @@ def run(  # noqa: PLR0913
 
         rule_state.reset_parameter_containers()
 
-        pbar_method: Callable = determine_progress_bar_method_by_environment()
```

Comment:
We don't care about RBP and I don't think it's worth adding a dep just to account for this